### DISCUSSION TITLE
Error on Time Tracking >> My Times page

### DISCUSSION THREAD

**Discussion by kfml** [kimai/kimai#3874] [General]
Error As Is:   There is one date input field and three time input fields.
Correct To Be:   There should be two dates input fields and two time input fields.

Could you please correct this asap? Thank you.

**Comment by kevinpapst** [comment]
Huh? There is a date and time for begin, then there is a duration field and a field for the calculated end time.
That is all you need. 

> Could you please correct this asap? Thank you.

There is nothing to fix. Especially "asap"...

**Comment by kfml** [comment]
Excuse me.
I will look at this again.
Maybe I overlooked something.
Excuse my mistake if this is the case.

Kevin

**Comment by kfml** [comment]
I user the interface has chnaged.
It is indeed working.
I thought it was not working.
Excise my error.

With the new updates I changed the theme to dark mode. I am finding it hard
to see the interface and would like to change it back.
I have looked under theme and cannot find the toggle switch the theme bavl
to the original bright/light mode.

Thanks for your help.

**Reply by kevinpapst** [reply]
Are you a Cloud customer? If so, please use the [Support channel on the internal part of www.kimai.cloud](https://www.kimai.cloud/go/profile_support) in the future and not this discussion forum for such questions. I need the context of knowing who you are and why you are writing, your initial posting makes much more sense now!

You can change the theme back in your own user preferences:
<img width="809" alt="Bildschirm­foto 2023-02-24 um 12 53 33" src="https://user-images.githubusercontent.com/533162/221173378-e4b47d1a-b179-4726-a28c-13e7e08c0368.png">

**Comment by kfml** [comment]
Thanks so so much.
Excuse my mistake earlier.

I will be recommending your system to others. Have a great weekend.
